skills/repo-indexer/references/templates.md

```
@@ -2,7 +2,10 @@
 
 ## Contents
 
-- [CLAUDE.md](#claudemd-500-tokens)
+- [CLAUDE.md (base)](#claudemd-500-tokens)
+- [CLAUDE.md — Monorepo](#claudemd-monorepo-variant)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)